.env

```diff
@@ -0,0 +1 @@
+GENERATE_SOURCEMAP=false
```

Comment:
What's this for? Also we probably should not be committing `.env` files to the repo

Reply:
I needed to add this to remove the like 30 plus warnings that were generating. Wasn't sure what other way to go about doing it honestly lol, this was just the solution I found on stack overflow

Reply:
removed .env, added .env.example